src/libstd/sys/common/unwind/gcc.rs

```diff
@@ -231,3 +231,31 @@ pub mod eabi {
         }
     }
 }
+
+#[cfg(all(target_os="windows", target_arch = "x86", target_env="gnu", not(test)))]
+#[lang = "eh_unwind_resume"]
+#[unwind]
+unsafe extern fn rust_eh_unwind_resume(panic_ctx: *mut u8) -> ! {
+    uw::_Unwind_Resume(panic_ctx as *mut uw::_Unwind_Exception);
```

Comment:
I didn't want to leave it up to the linker to choose which `_Unwind_Resume` this gets resolved to (what if `-lgcc_s` is added by the user?).
Having this as a lang item seems safer (also conceptually tidier, IMO, since this is a part of language runtime, just like eh_personality).